app/accuracy_app/accuracy_check.cpp

```diff
@@ -0,0 +1,23 @@
+#include <opencv2/opencv.hpp>
+#include <iostream>
+#include <filesystem>
+
+int main() {
+  std::string directory = "./photos";
+
+  for (const auto& entry : std::filesystem::directory_iterator(directory)) {
+    if (entry.is_regular_file()) {
+      cv::Mat image = cv::imread(entry.path().string());
+
+      // For example:
+      if (!image.empty()) {
+        cv::imshow("Image", image);
+        cv::waitKey(0);
```

Comment:
This will show the images in a sequential order and block the execution. It is better not to do this. Our future application will not do that, actually.

What you need to do at this stage: read the content of the image, convert to tensor (cv::Mat -> Tensor, use our tensor class) and do nothing with this tensor (for now)